lib/modules/manager/copier/artifacts.spec.ts

```diff
@@ -120,6 +120,27 @@ describe('modules/manager/copier/artifacts', () => {
           cmd: 'copier update --skip-answered --defaults --answers-file .copier-answers.yml --vcs-ref 1.1.0',
         },
       ]);
+      expect(execSnapshots[0].options?.cwd).toBe('/tmp/github/some/repo');
+    });
+
+    it('invokes copier update with nested destination and answer file', async () => {
+      const execSnapshots = mockExecAll();
+
+      await updateArtifacts({
+        packageFileName: 'apps/my-app/.copier-answers.yml',
+        updatedDeps: upgrades,
+        newPackageFileContent: '',
+        config: {},
+      });
+
+      expect(execSnapshots).toMatchObject([
+        {
+          cmd: 'copier update --skip-answered --defaults --answers-file .copier-answers.yml --vcs-ref 1.1.0',
+        },
+      ]);
+      expect(execSnapshots[0].options?.cwd).toBe(
+        '/tmp/github/some/repo/apps/my-app',
+      );
```

Comment:
```suggestion
          options: {
            cwd: '/tmp/github/some/repo/apps/my-app',
          }
        },
      ]);
```

use a single matcher